tidb.go

```diff
@@ -74,7 +73,9 @@ var (
 	domap = &domainMap{
 		domains: map[string]*domain.Domain{},
 	}
-	stores = make(map[string]kv.Driver)
+	stores    = make(map[string]kv.Driver)
```

Comment:
But we already has the `RegisterLocalStore` and `RegisterStore` for user to register the stores. As there still has some valid in these functions.